ProjectSettings/TagManager.asset

```diff
@@ -13,7 +13,7 @@ TagManager:
   - UI
   - 
   - 
-  - 
+  - SpatialProcessingPlane
```

Comment:
Not sure we should be adding anything to the tag manager manually. 

When people import this example, tags are generally not included.

Reply:
If that's the case, I guess developers have to manually add the layer?

Reply:
indeed, or we can hard code it.

Reply:
It's related to #1443

Reply:
I just read #1443. In this case, I think we can use layer 31:Spatial Mapping and add instruction to use Configure > Apply Mixed Reality Project Settings.
Thank you!

![2018-05-17 12_49_47-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200329-dc8cc4a4-59d0-11e8-8f23-05e0d06bdbbe.png)

![2018-05-17 12_49_40-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200330-dca1a7fc-59d0-11e8-988c-9e0339ad3d71.png)